packages/fclasses/src/flowHoc.tsx

```diff
@@ -40,9 +40,15 @@ function mergeMeta(objValue:any, srcValue:any) {
 const preserveMeta = (hoc: HOCBase): HOCBase => <P extends object, Q extends object = P>(
   Component: ComponentOrTag<P>,
 ): ComponentWithMeta<Q> => {
-  const NewComponent = hoc(Component) as ComponentWithMeta<Q>;
-  const finalMeta = mergeWith({}, Component, NewComponent, mergeMeta);
-  return Object.assign(NewComponent, finalMeta);
+  try {
+    const NewComponent = hoc(Component) as ComponentWithMeta<Q>;
+    const finalMeta = mergeWith({}, Component, NewComponent, mergeMeta);
+    return Object.assign(NewComponent, finalMeta);
+  } catch (e) {
+    console.log(e);
+    console.log(Component);
```

Comment:
More logs here.

Reply:
@lucas-varela , fixed as part of https://github.com/wodenx/Bodiless-JS/pull/23. Thanks!